crds/workspace.devfile.io_devworkspaces.v1beta1.yaml

```diff
@@ -5499,9 +5499,17 @@ spec:
                               - name
                               type: object
                             registryUrl:
+                              description: Registry URL to pull devfile from with
```

Comment:
Consider changing to:
"Registry URL to pull the parent devfile from when using `id` in the parent reference. To ensure the parent devfile gets resolved consistently in different environments, it is recommended to always specify the `regsitryURL` when `id` is used." 
(with `registryURL and `id` properly formated)

Reply:
updated